adwords/report.js

```diff
@@ -37,7 +37,7 @@ class AdwordsReport {
 
         apiVersion = apiVersion || AdwordsConstants.DEFAULT_ADWORDS_VERSION;
 
-        this.getHeaders((error, headers) => {
+        this.getHeaders(report.headers, (error, headers) => {
```

Comment:
can we use something like `additionalHeaders` instead of just headers. That makes it a bit more readable as it seems like you are passing in all headers here.